proto/tendermint/types/types.proto

```diff
@@ -87,9 +87,11 @@ message Data {
   // NOTE: not all txs here are valid.  We're just agreeing on the order first.
   // This means that block.AppHash does not include these txs.
   repeated bytes txs = 1;
-  IntermediateStateRoots intermediate_state_roots = 2 [(gogoproto.nullable) = false];
-  EvidenceData evidence    = 3 [(gogoproto.nullable) = false];
-  Messages messages        = 4 [(gogoproto.nullable) = false];
+
+  bytes meta_data = 2;
+  IntermediateStateRoots intermediate_state_roots = 3 [(gogoproto.nullable) = false];
+  EvidenceData evidence    = 4 [(gogoproto.nullable) = false];
+  Messages messages        = 5 [(gogoproto.nullable) = false];
```

Comment:
We won't need both `meta_data` and the added lazyledger specific fields here (`IntermediateStateRoots` and `Messages`).